e2e/cypress/support/external_commands.d.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/// <reference types="cypress" />
+
+// ***************************************************************
+// Each command should be properly documented using JSDoc.
+// See https://jsdoc.app/index.html for reference.
+// Basic requirements for documentation are the following:
+// - Meaningful description
+// - Each parameter with `@params`
+// - Return value with `@returns`
+// - Example usage with `@example`
+// Custom command should follow naming convention of having `visual` prefix, e.g. `visualEyesOpen`.
```

Comment:
```suggestion
// Custom command should follow naming convention of having `external` prefix, e.g. `externalAddUserToTeam`.
```

Reply:
@saturninoabril, Good catch. Fixed this. Please recheck.